independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/reader/QuteCodestartFileReader.java

```diff
@@ -6,9 +6,13 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.concurrent.CompletionStage;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 import org.apache.commons.io.FilenameUtils;
 
+import com.fasterxml.jackson.core.Version;
```

Comment:
I'm not a big fan of depending on Jackson here. `io.smallrye.common:smallrye-common-version` may be a better substitute here

Reply:
Fixed, used Maven one